forward/events.go

```diff
@@ -0,0 +1,64 @@
+package forward
+
+// An EventListener handles events given to it by the SWIM node. HandleEvent should be thread safe.
+type EventListener interface {
```

Comment:
I am a little bit confused about the terminology, but shouldn't this be an EventHandler?

I see an EventListener as something that you can register handlers on.